test/fuzz/auto-compounder/_AutoCompounder.fuzz.t.sol

```diff
@@ -4,54 +4,46 @@
  */
 pragma solidity 0.8.22;
 
-import { Fuzz_Test } from "../../../lib/accounts-v2/test/fuzz/Fuzz.t.sol";
+import { Fuzz_Test } from "../Fuzz.t.sol";
 
-import { ArcadiaOracle } from "../../../lib/accounts-v2/test/utils/mocks/oracles/ArcadiaOracle.sol";
-import { AutoCompounder } from "../../../src/auto-compounder/AutoCompounder.sol";
 import { AutoCompounderExtension } from "../../utils/extensions/AutoCompounderExtension.sol";
-import { AssetValueAndRiskFactors } from "../../../lib/accounts-v2/src/libraries/AssetValuationLib.sol";
-import { BitPackingLib } from "../../../lib/accounts-v2/src/libraries/BitPackingLib.sol";
-import { ERC20Mock, ERC20 } from "../../../lib/accounts-v2/test/utils/mocks/tokens/ERC20Mock.sol";
-import { FixedPointMathLib } from "../../../lib/accounts-v2/lib/solmate/src/utils/FixedPointMathLib.sol";
-import { FixedPoint96 } from "../../../lib/accounts-v2/src/asset-modules/UniswapV3/libraries/FixedPoint96.sol";
+import { AutoCompounder } from "../../../src/auto-compounder/AutoCompounder.sol";
+import { ERC20Mock } from "../../../lib/accounts-v2/test/utils/mocks/tokens/ERC20Mock.sol";
+import { UniswapV3AMExtension } from "../../../lib/accounts-v2/test/utils/extensions/UniswapV3AMExtension.sol";
+import { Utils } from "../../../lib/accounts-v2/test/utils/Utils.sol";
 import { IUniswapV3PoolExtension } from
     "../../../lib/accounts-v2/test/utils/fixtures/uniswap-v3/extensions/interfaces/IUniswapV3PoolExtension.sol";
-import { INonfungiblePositionManagerExtension } from
-    "../../../lib/accounts-v2/test/utils/fixtures/uniswap-v3/extensions/interfaces/INonfungiblePositionManagerExtension.sol";
-import {
-    SwapRouter02Fixture,
-    ISwapRouter02
-} from "../../../lib/accounts-v2/test/utils/fixtures/swap-router-02/SwapRouter02Fixture.f.sol";
-import { TickMath } from "../../../lib/accounts-v2/src/asset-modules/UniswapV3/libraries/TickMath.sol";
-import { UniswapV3Fixture } from "../../../lib/accounts-v2/test/utils/fixtures/uniswap-v3/UniswapV3Fixture.f.sol";
-import { Utils } from "../../../lib/accounts-v2/test/utils/Utils.sol";
+import { ISwapRouter02 } from "../../../lib/accounts-v2/test/utils/fixtures/swap-router-02/interfaces/ISwapRouter02.sol";
 
 /**
  * @notice Common logic needed by all "AutoCompounder" fuzz tests.
  */
-abstract contract AutoCompounder_Fuzz_Test is Fuzz_Test, UniswapV3Fixture, SwapRouter02Fixture {
-    using FixedPointMathLib for uint256;
+abstract contract AutoCompounder_Fuzz_Test is Fuzz_Test {
     /*////////////////////////////////////////////////////////////////
                             CONSTANTS
     /////////////////////////////////////////////////////////////// */
 
-    int24 public MAX_TICK_VALUE = 887_272;
```

Comment:
I just removed all hardcoded values, because they have to be fuzzed.
We are running in loops now.